app/wizard_pipelines/feedback/steps/rating.rb

```diff
@@ -0,0 +1,24 @@
+module Feedback
+  module Steps
+    class Rating < ::GITWizard::Step
+      RATINGS = [
+        OpenStruct.new(id: 0, value: "Very satisfied"),
+        OpenStruct.new(id: 1, value: "Somewhat satisfied"),
```

Comment:
@StackedPancakez We are recording this as `Satisfied` in the database so we are being inconsistent in users select to what we capture

I've checked the GDS manual and they use `Satisfied`  rather than `Somewhat satisfied` so please could you change the front end label to `Satisfied` (and keep the data capture in the database the same)